#0 building with "default" instance using docker driver

#1 [internal] load .dockerignore
#1 ...

#2 [internal] load build definition from Dockerfile
#2 transferring dockerfile: 403B 0.0s done
#2 DONE 0.4s

#1 [internal] load .dockerignore
#1 transferring context: 2B done
#1 DONE 0.5s

#3 [internal] load metadata for docker.io/library/alpine:latest
#3 ...

#4 [auth] library/alpine:pull token for registry-1.docker.io
#4 DONE 0.0s

#3 [internal] load metadata for docker.io/library/alpine:latest
#3 DONE 1.6s

#5 [1/7] FROM docker.io/library/alpine:latest@sha256:0a4eaa0eecf5f8c050e5bba433f58c052be7587ee8af3e8b3910ef9ab5fbe9f5
#5 resolve docker.io/library/alpine:latest@sha256:0a4eaa0eecf5f8c050e5bba433f58c052be7587ee8af3e8b3910ef9ab5fbe9f5
#5 resolve docker.io/library/alpine:latest@sha256:0a4eaa0eecf5f8c050e5bba433f58c052be7587ee8af3e8b3910ef9ab5fbe9f5 0.5s done
#5 DONE 0.5s

#6 [internal] load build context
#6 transferring context: 2.59MB 0.3s done
#6 DONE 0.6s

#5 [1/7] FROM docker.io/library/alpine:latest@sha256:0a4eaa0eecf5f8c050e5bba433f58c052be7587ee8af3e8b3910ef9ab5fbe9f5
#5 sha256:0a4eaa0eecf5f8c050e5bba433f58c052be7587ee8af3e8b3910ef9ab5fbe9f5 1.85kB / 1.85kB done
#5 sha256:eddacbc7e24bf8799a4ed3cdcfa50d4b88a323695ad80f317b6629883b2c2a78 528B / 528B done
#5 sha256:324bc02ae1231fd9255658c128086395d3fa0aedd5a41ab6b034fd649d1a9260 1.47kB / 1.47kB done
#5 sha256:c6a83fedfae6ed8a4f5f7cbb6a7b6f1c1ec3d86fea8cb9e5ba2e5e6673fde9f6 0B / 3.62MB 0.1s
#5 sha256:c6a83fedfae6ed8a4f5f7cbb6a7b6f1c1ec3d86fea8cb9e5ba2e5e6673fde9f6 3.62MB / 3.62MB 0.3s
#5 extracting sha256:c6a83fedfae6ed8a4f5f7cbb6a7b6f1c1ec3d86fea8cb9e5ba2e5e6673fde9f6
#5 sha256:c6a83fedfae6ed8a4f5f7cbb6a7b6f1c1ec3d86fea8cb9e5ba2e5e6673fde9f6 3.62MB / 3.62MB 0.4s done
#5 extracting sha256:c6a83fedfae6ed8a4f5f7cbb6a7b6f1c1ec3d86fea8cb9e5ba2e5e6673fde9f6 0.7s done
#5 DONE 2.2s

#7 [2/7] RUN apk add --no-cache python3 &&   python3 -m ensurepip &&   rm -r /usr/lib/python*/ensurepip &&   pip3 install --upgrade pip setuptools &&   rm -r /root/.cache
#7 1.440 fetch https://dl-cdn.alpinelinux.org/alpine/v3.20/main/x86_64/APKINDEX.tar.gz
#7 2.452 fetch https://dl-cdn.alpinelinux.org/alpine/v3.20/community/x86_64/APKINDEX.tar.gz
#7 3.989 (1/17) Installing libbz2 (1.0.8-r6)
#7 4.002 (2/17) Installing libexpat (2.6.2-r0)
#7 4.016 (3/17) Installing libffi (3.4.6-r0)
#7 4.028 (4/17) Installing gdbm (1.23-r1)
#7 4.048 (5/17) Installing xz-libs (5.6.1-r3)
#7 4.074 (6/17) Installing libgcc (13.2.1_git20240309-r0)
#7 4.088 (7/17) Installing libstdc++ (13.2.1_git20240309-r0)
#7 4.243 (8/17) Installing mpdecimal (4.0.0-r0)
#7 4.260 (9/17) Installing ncurses-terminfo-base (6.4_p20240420-r0)
#7 4.298 (10/17) Installing libncursesw (6.4_p20240420-r0)
#7 4.316 (11/17) Installing libpanelw (6.4_p20240420-r0)
#7 4.332 (12/17) Installing readline (8.2.10-r0)
#7 4.343 (13/17) Installing sqlite-libs (3.45.3-r1)
#7 4.391 (14/17) Installing python3 (3.12.3-r1)
#7 5.301 (15/17) Installing python3-pycache-pyc0 (3.12.3-r1)
#7 5.919 (16/17) Installing pyc (3.12.3-r1)
#7 5.929 (17/17) Installing python3-pyc (3.12.3-r1)
#7 5.941 Executing busybox-1.36.1-r29.trigger
#7 5.973 OK: 50 MiB in 31 packages
#7 15.20 error: externally-managed-environment
#7 15.20 
#7 15.20 × This environment is externally managed
#7 15.20 ╰─> 
#7 15.20     The system-wide python installation should be maintained using the system
#7 15.20     package manager (apk) only.
#7 15.20     
#7 15.20     If the package in question is not packaged already (and hence installable via
#7 15.20     "apk add py3-somepackage"), please consider installing it inside a virtual
#7 15.20     environment, e.g.:
#7 15.20     
#7 15.20     python3 -m venv /path/to/venv
#7 15.20     . /path/to/venv/bin/activate
#7 15.20     pip install mypackage
#7 15.20     
#7 15.20     To exit the virtual environment, run:
#7 15.20     
#7 15.20     deactivate
#7 15.20     
#7 15.20     The virtual environment is not deleted, and can be re-entered by re-sourcing
#7 15.20     the activate file.
#7 15.20     
#7 15.20     To automatically manage virtual environments, consider using pipx (from the
#7 15.20     pipx package).
#7 15.20 
#7 15.20 note: If you believe this is a mistake, please contact your Python installation or OS distribution provider. You can override this, at the risk of breaking your Python installation or OS, by passing --break-system-packages.
#7 15.20 hint: See PEP 668 for the detailed specification.
#7 15.63 Traceback (most recent call last):
#7 15.63   File "<frozen runpy>", line 198, in _run_module_as_main
#7 15.63   File "<frozen runpy>", line 88, in _run_code
#7 15.63   File "/usr/lib/python3.12/ensurepip/__main__.py", line 5, in <module>
#7 15.63     sys.exit(ensurepip._main())
#7 15.63              ^^^^^^^^^^^^^^^^^
#7 15.63   File "/usr/lib/python3.12/ensurepip/__init__.py", line 284, in _main
#7 15.63     return _bootstrap(
#7 15.63            ^^^^^^^^^^^
#7 15.63   File "/usr/lib/python3.12/ensurepip/__init__.py", line 200, in _bootstrap
#7 15.63     return _run_pip([*args, *_PACKAGE_NAMES], additional_paths)
#7 15.63            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
#7 15.63   File "/usr/lib/python3.12/ensurepip/__init__.py", line 101, in _run_pip
#7 15.63     return subprocess.run(cmd, check=True).returncode
#7 15.63            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
#7 15.63   File "/usr/lib/python3.12/subprocess.py", line 571, in run
#7 15.63     raise CalledProcessError(retcode, process.args,
#7 15.63 subprocess.CalledProcessError: Command '['/usr/bin/python3', '-W', 'ignore::DeprecationWarning', '-c', '\nimport runpy\nimport sys\nsys.path = [\'/tmp/tmpw4lk8fys/pip-24.0-py3-none-any.whl\'] + sys.path\nsys.argv[1:] = [\'install\', \'--no-cache-dir\', \'--no-index\', \'--find-links\', \'/tmp/tmpw4lk8fys\', \'pip\']\nrunpy.run_module("pip", run_name="__main__", alter_sys=True)\n']' returned non-zero exit status 1.
#7 ERROR: process "/bin/sh -c apk add --no-cache python3 &&   python3 -m ensurepip &&   rm -r /usr/lib/python*/ensurepip &&   pip3 install --upgrade pip setuptools &&   rm -r /root/.cache" did not complete successfully: exit code: 1
------
 > [2/7] RUN apk add --no-cache python3 &&   python3 -m ensurepip &&   rm -r /usr/lib/python*/ensurepip &&   pip3 install --upgrade pip setuptools &&   rm -r /root/.cache:
15.63            ^^^^^^^^^^^
15.63   File "/usr/lib/python3.12/ensurepip/__init__.py", line 200, in _bootstrap
15.63     return _run_pip([*args, *_PACKAGE_NAMES], additional_paths)
15.63            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
15.63   File "/usr/lib/python3.12/ensurepip/__init__.py", line 101, in _run_pip
15.63     return subprocess.run(cmd, check=True).returncode
15.63            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
15.63   File "/usr/lib/python3.12/subprocess.py", line 571, in run
15.63     raise CalledProcessError(retcode, process.args,
15.63 subprocess.CalledProcessError: Command '['/usr/bin/python3', '-W', 'ignore::DeprecationWarning', '-c', '\nimport runpy\nimport sys\nsys.path = [\'/tmp/tmpw4lk8fys/pip-24.0-py3-none-any.whl\'] + sys.path\nsys.argv[1:] = [\'install\', \'--no-cache-dir\', \'--no-index\', \'--find-links\', \'/tmp/tmpw4lk8fys\', \'pip\']\nrunpy.run_module("pip", run_name="__main__", alter_sys=True)\n']' returned non-zero exit status 1.
------
Dockerfile:3
--------------------
   2 |     
   3 | >>> RUN apk add --no-cache python3 && \
   4 | >>>   python3 -m ensurepip && \
   5 | >>>   rm -r /usr/lib/python*/ensurepip && \
   6 | >>>   pip3 install --upgrade pip setuptools && \
   7 | >>>   rm -r /root/.cache
   8 |     
--------------------
ERROR: failed to solve: process "/bin/sh -c apk add --no-cache python3 &&   python3 -m ensurepip &&   rm -r /usr/lib/python*/ensurepip &&   pip3 install --upgrade pip setuptools &&   rm -r /root/.cache" did not complete successfully: exit code: 1
Duration: 0:21.91